conf/app.routes

```diff
@@ -16,8 +16,6 @@
 
 GET         /                                                                           controllers.IndexController.onPageLoad()
 
-# upload file notification callback
-#NOCSRF
```

Comment:
@westwater if you think this is needed please let us know and we can have a chat about it